steering_controllers_library/doc/userdoc.rst

```diff
@@ -82,9 +83,16 @@ With the following state interfaces:
 Subscribers
 ,,,,,,,,,,,,
 
-Used when controller is not in chained mode (``in_chained_mode == false``).
-
+Used when controller is not in chained mode (``in_chained_mode == false``) and the twist input mode is activated (``twist_input == true``):
 - ``<controller_name>/reference``  [`geometry_msgs/msg/TwistStamped <twist_msg_>`_]
+In this configuration the controller uses :
+- **Linear Velocity (`linear`)**: Represents the linear speed of the robot (in meters per second, m/s).
+- **Angular Velocity (`angular`)**: Represents the angular speed of the robot (in meters per second, m/s).
```

Comment:
```suggestion
- **Linear Velocity (`linear`)**: Represents the linear velocity of the robot (in meters per second, m/s).
- **Angular Velocity (`angular`)**: Represents the angular velocity of the robot (in meters per second, m/s).
```

I'm not a native English speaker, but how I always saw this: `speed = | velocity |`, speed does not have a sign.